packages/admin/cms-admin/src/generator/future/generateGrid.ts

```diff
@@ -248,6 +248,21 @@ export function generateGrid(
 
         let gridType: "number" | "boolean" | "dateTime" | "date" | undefined;
 
+        let filterOperators: string | undefined;
+        if (column.filterOperators) {
+            let importPath = column.filterOperators.import;
+            if (importPath.startsWith("./")) {
+                //go one level up as generated files are in generated subfolder
+                importPath = `.${importPath}`;
+            }
```

Comment:
Do we perform this trick for all import statements?

Reply:
as far as I know this is only used for ImportReference-type in config, like it's used in form-generator to generate a reference to field validation-code. probably also for generated graphql-query-files, but I would need to check this... (I copied this from [here](https://github.com/vivid-planet/comet/blob/main/packages/admin/cms-admin/src/generator/future/generateForm/generateFormField.ts#L62))

Reply:
I see... then it's okay 🙂